addons/website/static/src/builder/plugins/options/table_of_content_option_plugin_translate.js

```diff
@@ -0,0 +1,40 @@
+import { Plugin } from "@html_editor/plugin";
+import { applyFunDependOnSelectorAndExclude } from "../utils";
+
+export class TableOfContentOptionPlugin extends Plugin {
```

Comment:
You should add the notion of translate in the name or we will try to use the other TableOfContentOptionPlugin